x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/schema.ts

```diff
@@ -6,7 +6,13 @@
  */
 
 import type { MakeSchemaFrom } from '@kbn/usage-collection-plugin/server';
-import type { CspmUsage } from './types';
+import type { CspmUsage, ResourceStats } from './types';
+
+const byResourceType: MakeSchemaFrom<ResourceStats[string]> = {
```

Comment:
```suggestion
const resourceTypeSchema: MakeSchemaFrom<ResourceStats[string]> = {
```

i'd prefer if we keep `schema` in the name, like we have for others (`cspmUsageSchema`)